content/en/docs/components/katib/katib-config.md

```diff
@@ -0,0 +1,211 @@
+
+You can edit this Config Map even after deploying Katib.
+
+If you deploy Katib in Kubeflow namespace, to edit Katib config run this:
+
+`kubectl edit configMap katib-config -n kubeflow`
+
+## Metrics Collector Sidecar settings
+
+These settings are related to Katib metrics collectors, where:
+
+- key = `metrics-collector-sidecar`
+- value = corresponding JSON settings for each metrics collector kind.
+
+Example for the `File` metrics collector with all settings:
+
+```json
+metrics-collector-sidecar: |-
+{
+  "File": {
+    "image": "docker.io/kubeflowkatib/file-metrics-collector",
+    "imagePullPolicy": "Always",
+    "resources": {
+      "requests": {
+        "memory": "200Mi",
+        "cpu": "250m",
+        "ephemeral-storage": "200Mi"
+      },
+      "limits": {
+        "memory": "1Gi",
+        "cpu": "500m",
+        "ephemeral-storage": "2Gi"
+      }
+    }
+  },
+  ...
+}
+```
+
+All of these settings except **`image`** can be omitted. If you don't specify any other settings, default value is set.
+
+1. `image` - Docker image for the `File` metrics collector's container.
+
+   **Must be specified**.
+
+1. `imagePullPolicy` - [Image pull policy](https://kubernetes.io/docs/concepts/configuration/overview/#container-images) for the `File` metrics collector's container.
+
+   Default value is `IfNotPresent`.
+
+1. `resources` - [Resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container)
+   for the `File` metrics collector's container. In the above example you can see how to specify
+   `limits` and `requests`. Currently, you can specify only `memory`, `cpu` and `ephemeral-storage` resource.
+
+   Default values for the `requests` are:
+
+   - `memory = 10Mi`.
+   - `cpu = 50m`.
+   - `ephemeral-storage = 500Mi`.
+
+   Default values for the `limits` are:
+
+   - `memory = 100Mi`.
+   - `cpu = 500m`.
+   - `ephemeral-storage = 5Gi`.
+
+## Suggestion settings
+
+These settings are related to Katib suggestions, where:
+
+- key = `suggestion`
+- value = corresponding JSON settings for each algorithm name.
+
+If you want to use new algorithm, you must update Katib config with the new suggestion.
+
+Example for the `random` algorithm with all settings:
+
+```json
+suggestion: |-
+{
+  "random": {
+    "image": "docker.io/kubeflowkatib/suggestion-hyperopt",
+    "imagePullPolicy": "Always",
+    "resources": {
+      "requests": {
+        "memory": "100Mi",
+        "cpu": "100m",
+        "ephemeral-storage": "100Mi"
+      },
+      "limits": {
+        "memory": "500Mi",
+        "cpu": "500m",
+        "ephemeral-storage": "3Gi"
+      }
+    },
+    "serviceAccountName": "random-sa"
+  },
+  ...
+}
+```
+
+All of these settings except **`image`** can be omitted.
+If you don't specify any other settings, default value is set.
+
+1. `image` - Docker image for the suggestion's container with `random` algorithm.
+
+   **Must be specified**.
```

Comment:
```suggestion
   **The following values must be specified**:
```

(Does this suggestion make sense?)

Reply:
See comment: https://github.com/kubeflow/website/pull/2312#discussion_r516328626.